server/models/Request.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema({
+  user_id: {
+    type: String,
+    required: true
+  },
+  sitter_id: {
+    type: String,
+    required: true
+  },
```

Comment:
This is fine for now, however, when we have a `User` model we would prefer to have this as a reference (see [Mongoose `populate`](https://mongoosejs.com/docs/populate.html). It might be a good idea to leave a `TODO` comment for these?

Reply:
Sitter ID will be referencing the Profile which is in progress, when that is megred in @ar2rworld I'll get you to refactor this file to reference it. No need to add in `TODO` comments 😄